setup.py

```diff
@@ -25,18 +26,18 @@ def get_test_suite():
     long_description = f.read()
 
 setup(
-    name='linode-api',
+    name='linode_api4',
 
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='4.1.9b1',
+    version='1.0.0',
```

Comment:
do you want to make this 1.0.0b0 as suggested in the PR commit message?

Reply:
Nope, this is not a beta release